README.md

```diff
@@ -8,3 +8,28 @@
 
 ## Starting the Server
 1. In the root directory, type `npm run start`
+
+
+## CRUD routes
+GET A SELLER & PRODUCTS:
+  Get('/api/seller/:id')
+
+GET A STORE WITH SELLER AND PRODUCTS:
+  Get('/api/store/:id')
+
+DELETE A STORE WITH SELLER AND PRODUCTS:
+  Delete('/api/store/:id')
+
+UPDATE A STORE NAME:
+  Put('/api/store/:id) with body ex:
+  "${store_name}" : "Pizza Hut"
+
```

Comment:
Nice! 
For the routes, you may want to include what the call returns (or the shape of the data that is returned)